proguard/proguard-rxjava2.pro

```diff
@@ -0,0 +1,19 @@
+-keepclassmembers class rx.internal.util.unsafe.*ArrayQueue*Field* {
+   long producerIndex;
+   long consumerIndex;
+}
+
+-keepclassmembers class rx.internal.util.unsafe.BaseLinkedQueueProducerNodeRef {
+    rx.internal.util.atomic.LinkedQueueNode producerNode;
+}
+
+-keepclassmembers class rx.internal.util.unsafe.BaseLinkedQueueConsumerNodeRef {
+    rx.internal.util.atomic.LinkedQueueNode consumerNode;
+}
+
+-dontnote rx.internal.util.PlatformDependent
+
+-keep class io.reactivex.Scheduler
+-keep class io.reactivex.Observer
+-keep class io.reactivex.disposables.Disposable
+-keep class io.reactivex.Observable
```

Comment:
maybe rxjava2 is not required proguard rules.

https://github.com/ReactiveX/RxJava/issues/4786